persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -198,6 +199,16 @@ CREATE TABLE MessageRecipientFailure (
       PRIMARY KEY (message_id, conversation_id, recipient_failure_type)
 );
 
+CREATE TABLE ConversationVerificationDegraded (
```

Comment:
why we need a new table?

Reply:
to store the protocol on wich status degraded. 
Compared to the current web i noticed that we might need to store also the user who is "guilty" of conversation degrading.